esy.lock.json

```diff
@@ -1,1719 +0,0 @@
-{
```

Comment:
why this file was removed?

Reply:
I didnt intend for the rest of that to go in there. Just the new example. I was otherwise messing with the project to see what happens if I update with `@opam/reason` dep because compiling throws an error. I will remove all the non-sample related commits.

Reply:
Please take a look at it now. I reverted to the original pull request.